watertap/property_models/seawater_prop_pack.py

```diff
@@ -246,7 +249,8 @@ def build(self):
             doc="Dynamic viscosity parameter 3 for term B",
         )
 
-        # diffusivity parameters, eq 6 in Bartholomew
+        # diffusivity parameters, 25 C
```

Comment:
Could we consider incorporating temperature-dependent diffusivity into the seawater property package, similar to what's done in the NaCl_T_dep_prop package? It looks like we're using NaCl diffusivity approaches for seawater by substituting TDS for NaCl, but we're not adjusting for temperature changes. The impact of temperature on diffusivity is pronounced in systems with large temperature gradients like MD.

Reply:
I don't recall if the MIT thermodynamic library, the main source of the seawater prop pack, has temperature-dependent diffusivity. Also, our solute is "TDS" in this prop pack, so deciding on what the effective diffusivity should be, especially as a function of temperature, might take some thought. One approach could be to consider the composition from Millero (that was used to estimate molecular weight value) and derive some relationship via something like PHREEQC. An easier approach could be just using the relationship from NaCl with temp dependence here too as an optional calculation for temp-dependent diffusivity. That said, I haven't looked at the lit recently to see if there is a more obvious approach. 

Any ideas?

Reply:
The MIT thermodynamic library does not have this property. I agree that one approach could be to consider the composition from Millero and derive a relationship using PHREEQC. I will look into the literature to see if this has been done or if there is a better approach. 

I think as a band-aid fix for now we could replace the current calculation with the relationship from the NaCl with temperature dependence property package. This would at least represent the impact of temperature.